test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Infrastructure/ApplicationTestFixture.cs

```diff
@@ -81,40 +68,35 @@ public virtual DeploymentParameters GetDeploymentParameters()
 #else
 #error the target framework needs to be updated.
 #endif
+#if DEBUG
+                Configuration = "Debug",
```

Comment:
Created a follow up item for this - https://github.com/aspnet/MvcPrecompilation/issues/99. Checking this in since the tests in the current form sometimes fail in the CI.